nautobot/dcim/tests/test_migrations.py

```diff
@@ -1,14 +1,15 @@
+import uuid
 from unittest import skipIf
 
 from django.db import connection
 
 from nautobot.core.tests.test_migration import NautobotDataMigrationTest
 from nautobot.circuits.choices import CircuitTerminationSideChoices
-from nautobot.extras.choices import CustomFieldTypeChoices, RelationshipTypeChoices
+from nautobot.extras.choices import CustomFieldTypeChoices, ObjectChangeActionChoices, RelationshipTypeChoices
 
 
 class SiteAndRegionDataMigrationToLocation(NautobotDataMigrationTest):
-    migrate_from = [("dcim", "0029_change_tree_manager_on_tree_models")]
+    migrate_from = [("dcim", "0029_add_tree_managers_and_foreign_keys_pre_data_migration")]
```

Comment:
Is this going to continue to work as more migrations get added to `dcim`?

Reply:
Yeah as long as the `migrate_from` and `migrate_to` filenames do not change, it should work fine.